examples/template.html

```diff
@@ -59,7 +59,7 @@ <h1 class="as-subheader">Basic example template</h1>
   </div>
 
   <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js"></script>
-  <script src="https://unpkg.com/deck.gl@8.1.7/dist.min.js"></script>
+  <script src="https://unpkg.com/deck.gl@8.2.0-alpha.2/dist.min.js"></script>
```

Comment:
Great, please also add the reference to the description in the body: <li class="as-list__item">DeckGL 8.1.7</li>